src/main/java/core/basesyntax/entity/Circle.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.entity;
+
+public class Circle extends Figure {
+    public static final double MAX_RADIUS = 25;
+    public static final double DEFAULT_RADIUS = 10;
+    private double radius;
+
+    public Circle(Color color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public double getRadius() {
+        return radius;
+    }
+
+    public void setRadius(double radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public void printInfo() {
+        System.out.println(new StringBuilder("Figure:")
+                .append(" Circle,")
+                .append(System.lineSeparator())
+                .append("        area: ")
+                .append(getArea())
+                .append(" sq.units,")
+                .append(System.lineSeparator())
+                .append("        radius: ")
+                .append(radius)
+                .append(" units,")
+                .append(System.lineSeparator())
+                .append("        color: ")
+                .append(getColor()));

```

Comment:
You can use simple concatenation for these proposes, `StringBuilder` more suitable in runtime, for example when you have some loop and cases like that.